src/OrchardCore/OrchardCore.DisplayManagement/PositionWrapper.cs

```diff
@@ -50,5 +51,24 @@ public ValueTask<IShape> AddAsync(object item, string position)
         {
             throw new System.NotImplementedException();
         }
+
+        public static PositionWrapper TryWrap(object value, string position)
+        {
+            if (value is PositionWrapper wrapper)
+            {
+                // Update the new Position
+                wrapper.Position = position;
+                return wrapper;
+            }
+            else if (value is IHtmlContent content)
+                return new PositionWrapper(content, position);
+            else if (value is string stringContent)
+                return new PositionWrapper(stringContent, position);
+            else
+                throw new System.NotSupportedException($"Type of {nameof(value)} must be either {nameof(String)}, {nameof(IHtmlContent)} or {nameof(PositionWrapper)}.");
```

Comment:
`TryWrap` should it return a new wrapper in this case instead of throwing?

Reply:
Should it unwrap if the position is "" ? Is that the meaning of "" in ThemeLayout.cs?

Reply:
> `TryWrap` should it return a new wrapper in this case instead of throwing?

Throwing `NotSupportedException` because `value` is not either string, `IHtmlContent` or `PositionWrapper`.